src/Fetch.ml

```diff
@@ -216,6 +218,14 @@ let decodeRequestRedirect = (* internal *)
   | "manual" -> Manual
   | e -> raise (Failure ("Unknown requestRedirect: " ^ e))
 
+module AbortController = struct
+  type t = abortController
+  
+  external getSignal : t -> signal = "signal" [@@bs.get]
```

Comment:
Why `getSignal` instead of just `signal`, which would be consistent with both the JS API and all the other getters here?